scripts/build/setup/termux_setup_cmake.sh

```diff
@@ -25,8 +25,8 @@ termux_setup_cmake() {
 
 		export PATH=$TERMUX_CMAKE_FOLDER/bin:$PATH
 	else
-		if [[ "$(dpkg --version 2>/dev/null)" && "$(dpkg-query -W -f '${db:Status-Status}\n' cmake 2>/dev/null)" != "installed" ]] ||
-                   [[ "$(pacman -V 2>/dev/null)" && ! "$(pacman -Q cmake 2>/dev/null)" ]]; then
+		if [[ "$TERMUX_MAIN_PACKAGE_FORMAT" = "debian" && "$(dpkg-query -W -f '${db:Status-Status}\n' cmake 2>/dev/null)" != "installed" ]] ||
```

Comment:
So with these changes TERMUX_MAIN_PACKAGE_FORMAT is used in scripts/build and TERMUX_PACKAGE_FORMAT in packages/?

Can't the same parameter be used?

Reply:
No, because TERMUX_PACKAGE_FORMAT can be changed via --format, this may cause compilation errors.